doc/modules/multiclass.rst

```diff
@@ -109,12 +109,12 @@ One-Vs-The-Rest
 ===============
 
 This strategy, also known as **one-vs-all**, is implemented in
-:class:`OneVsRestClassifier`.  The strategy consists in fitting one classifier
+:class:`OneVsRestClassifier`.  The strategy consists of fitting one classifier
```

Comment:
not sure. http://english.stackexchange.com/questions/95929/consists-of-vs-consists-in-different-meanings-of-the-verb-or-the-same-mea http://english.stackexchange.com/questions/61600/consist-in-vs-consist-of

I think it is telling that the first hits for "consists in" are of stackexchange discussing the usage of the phrase.